crates/iracing-data-api-client/docs/SeriesApi.md

```
@@ -0,0 +1,206 @@
+
+## get_series_past_seasons
+
+> models::IracingApiResponse get_series_past_seasons(series_id)
+
+
+### Parameters
+
+
+Name | Type | Description  | Required | Notes
+------------- | ------------- | ------------- | ------------- | -------------
+**series_id** | **f64** |  | [required] |
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)